agent_deploy/kubernetes/sysdig-image-analyzer-daemonset-v1.yaml

```diff
@@ -0,0 +1,142 @@
+# apiVersion: extensions/v1beta1  # If you are in Kubernetes version 1.8 or less please use this line instead of the following one
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: sysdig-image-analyzer
+  labels:
+    app: sysdig-image-analyzer
+spec:
+  selector:
+    matchLabels:
+      app: sysdig-image-analyzer
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: sysdig-image-analyzer
+    spec:
+      volumes:
+      # Needed for cri-o image inspection.
+      # cri-o and especially OCP 4.x by default use containers/storage to handle images, and this makes sure that the
+      # analyzer has access to the configuration. This file is mounted read-only.
+      - name: etccontainersstorage-vol
+        hostPath:
+          path: /etc/containers/storage.conf
+      # Needed for cri-o image inspection.
+      # This is the directory where image data is stored by default when using cri-o and OCP 4.x and the analyzer
+      # uses it to get the data to scan. This directory must be mounted r/w because proper access to its files through
+      # the containers/storage library is always regulated with a lockfile.
+      - name: varlibcontainers-vol
+        hostPath:
+          path: /var/lib/containers
+      # Needed for socket access
+      - name: varrun-vol
+        hostPath:
+          path: /var/run
+      # Add custom volume here
+      - name: sysdig-image-analyzer-config
+        configMap:
+          name: sysdig-image-analyzer
+          optional: true
+      tolerations:
+        - effect: NoSchedule
+          key: node-role.kubernetes.io/master
+      # The following line is necessary for RBAC
+      serviceAccount: sysdig-agent
+      terminationGracePeriodSeconds: 5
+      containers:
+      - name: sysdig-image-analyzer
+        image: docker.io/sysdig/node-image-analyzer:latest
+        securityContext:
+          # needed for the CRI socket
+          privileged: true
```

Comment:
Could this have a much bigger WARNING or comment that this is not needed unless you are running ContainerD?